recipes/openssl/1.x.x/conanfile.py

```diff
@@ -758,9 +759,58 @@ def package(self):
 
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
+        self._create_cmake_module_variables(
+            os.path.join(self.package_folder, self._module_subfolder, self._module_file)
+        )
+
+    @staticmethod
+    def _create_cmake_module_variables(module_file):
+        content = """\
+            if(OpenSSL_FOUND)
+                set(OPENSSL_FOUND ${OpenSSL_FOUND})
+            endif()
```

Comment:
Maybe we should use `if(DEFINED VAR)` syntax here?

Because, if for whatever reason `OpenSSL_FOUND` is set, I think it makes sense to set `OPENSSL_FOUND` to the same value (it can be `True` or `False` or whatever, it doesn't matter).
And if it's not set, we won't set the uppercase variable as well.

And this comment works for other if conditions as well.
Please, share your opinion on this.

Reply:
if OpenSSL_FOUND is set, it is always TRUE. But yes let's use DEFINED, it should be neater.